spec/features/admin/overview_spec.rb

```diff
@@ -123,5 +110,45 @@
         end
       end
     end
+
+    context "with the spree dash configured" do
+      let(:d1) { create(:distributor_enterprise) }
+
+      before do
+        stub_jirafe
+        @enterprise_user.enterprise_roles.build(enterprise: d1).save
+      end
+
+      around do |example|
+        with_dash_configured { example.run }
```

Comment:
Does this mean the error is only on staging, because Dash is configured there, but not on production?